src/chapters/how-programs-work/replit.rst

```diff
@@ -1,48 +1,47 @@
-Repl.it
-=======
+Replit.com
+==========
 
-`Repl.it <https://repl.it>`__ and `Trinket.io <https://trinket.io>`__ share
+`Replit.com <https://replit.com>`__ and `Trinket.io <https://trinket.io>`__ share
 many of the same features, but the two sites differ in how they present their
 workspaces.
 
-The previous page explored Trinket, so now let's take a look at Repl.it.
+The previous page explored Trinket, so now let's take a look at Replit.com
 
-Create a Repl.it Account
--------------------------
+Create a Replit.com Account
+---------------------------
 
 .. admonition:: Warning
 
-   Before signing up for a repl.it account, be sure your school has approved
+   Before signing up for a replit.com account, be sure your school has approved
    the site for student use!
 
-Create a new student account with `Repl.it <https://repl.it/signup>`__.
+Create a new student account with `Replit.com <https://replit.com/signup>`__.
 
 .. figure:: figures/replit-signup.png
-   :alt: The repl.it sign-up screen.
+   :alt: The Replit.com sign-up screen.
 
 After you sign in, you will see your *Home* screen, which displays your recent
 work and announcements. Since you are just starting out, you won't need to do
 much on this screen.
 
 .. figure:: figures/replit-home.png
-   :alt: Repl.it home screen.
+   :alt: Replit.com home screen.
+Home screen
 
-   Home screen
-
-Click on *new repl* to begin a new project. Scroll through the options and
-select *Python*. Next, name your project and click *Create Repl*.
+Click on *new repl (1) or (2)* to begin a new project. Scroll through the options and
+select *Python (3)*. Next, name your project (4) and click *Create Repl (5)*.
 
 .. figure:: figures/replit-newrepl.png
    :alt: Create a new Python repl.
 
-The Repl.it Workspace
----------------------
+The Replit.com Workspace
+------------------------
 
-Now let's take a look at how to use Repl.it. The workspace consists of three
+Now let's take a look at how to use Replit. The workspace consists of three
 main panels and several menu functions.
 
 .. figure:: figures/replit-overview.png
-   :alt: Repl.it code editor layout
+   :alt: Replit.com code editor layout
```

Comment:
I like the new screenshot, but here's a few tweaks:
1. Trim the excess empty space (from the check mark down).  As much as possible, we want the kids to see a figure and its related text without needing to scroll.
2. Let's increase the size of the number labels.
3. Move the red labels outside of the figure.  This prevents the red/black overlap with the (3), and it allows us to better align the (6) with the three horizontal bars.

Reply:
Done